File: CTProyecto/app/src/main/java/pe/edu/upeu/ctproyecto/data/model/ZonaTuristica.kt
package pe.edu.upeu.ctproyecto.data.model

import com.google.gson.annotations.SerializedName

data class ZonaTuristica(
    @SerializedName("id_zona") val id: Int,
    val nombre: String,
    val descripcion: String?,
    val ubicacion: String?,
    val imagen_url: String?,
    val estado: String
)

File: WebTurismo/app/Models/ZonaTuristica.php
<?php

namespace App\Models;



use Illuminate\Database\Eloquent\Factories\HasFactory;
use Illuminate\Database\Eloquent\Model;


class ZonaTuristica extends Model
{
    use HasFactory;
    protected $table = 'zonas_turisticas';
    protected $primaryKey = 'id_zona';
    public $timestamps = false;

    protected $fillable = [
        'nombre',
        'descripcion',
        'ubicacion',
        'imagen_url',
        'estado',
        'fecha_creacion'
    ];
}
